app/javascript/schools/courses/components/curriculum_editor/CurriculumEditor__TargetChecklistItemEditor.res

```diff
@@ -39,13 +39,13 @@ let checklistDropdown = (checklistItem, updateChecklistItemCB) => {
   | LongText => "blue"
   | ShortText => "orange"
   | Files => "pink"
```

Comment:
Colors like Pink and Teal removed from the theme. Need to replace them with other available colors.